R/fit.R

```diff
@@ -441,15 +441,15 @@ mmrm <- function(formula,
     drop_visit_levels = control$drop_visit_levels,
     allow_na_response = FALSE
   )
-  fit <- list()
+  fit <- structure("", class = "try-error")
```

Comment:
are we sure we want to change this here?

Reply:
<img width="529" alt="image" src="https://github.com/openpharma/mmrm/assets/36079400/fa636206-4ab2-4742-a035-e5f64626f508">

from this part it has to be "try-error" to consistent, or change this function to use length as error indicator

Reply:
(and this will affect more docs and unit tests, just because previously it is all taken as "try-error")

Reply:
ok sounds good